### ISSUE TITLE
Build failure: missing mtl-compat dep

### ISSUE THREAD

**Issue by snoyberg** [brendanhay/amazonka#89]
Last Stackage build failure:

```
Unpacking to amazonka-0.3.3/
Resolving dependencies...
Configuring amazonka-0.3.3...
Building amazonka-0.3.3...
Preprocessing library amazonka-0.3.3...

src/Control/Monad/Trans/AWS.hs:107:18:
    Could not find module ‘Control.Monad.Except’
    It is a member of the hidden package ‘mtl-compat-0.2.1.3’.
    Perhaps you need to add ‘mtl-compat’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
```

**Comment by brendanhay** [comment]
We appear to have heisen-builds going on locally. Thanks!

**Comment by brendanhay** [comment]
Slightly confusing that Hackage pulled in ` mtl-2.2.1` and that module OK: http://hackage.haskell.org/package/amazonka-0.3.3/reports/1

I'll remove `0.3.3` from the preferred version range on Hackage (does that even work?) and make `amazonka-0.3.3.1` available.

**Comment by brendanhay** [comment]
Fixed by 64bbf31.
